ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/InMemoryKVParticipantState.scala

```diff
@@ -553,7 +553,7 @@ class InMemoryKVParticipantState(
   /** Get a new record time for the ledger from the system clock.
     * Public for use from integration tests.
     */
-  def getNewRecordTime(): Timestamp =
+  def getNewRecordTime: Timestamp =
```

Comment:
`()` many times serves as a useful reminder that the method performs side effect. If you don't have a strong opinion about it I would prefer to keep it.

Reply:
Seconded; if nothing else, it tells you can't easily inline the result. It's not exactly `IO[Timestamp]` but it's something.

Reply:
Perhaps rename it to simply `newRecordTime` to stop IntelliJ complaining?